commands/local_server_stop.go

```diff
@@ -36,17 +39,49 @@ var localServerStopCmd = &console.Command{
 	Usage:    "Stop the local web server",
 	Flags: []console.Flag{
 		dirFlag,
+		&console.BoolFlag{Name: "all", Usage: "Stop all local web servers"},
 	},
 	Action: func(c *console.Context) error {
+		if c.Bool("all") && c.IsSet("dir") {
+			return fmt.Errorf("you cannot use the --all option with a specific directory")
+		}
+
+		var dirs []string
 		projectDir, err := getProjectDir(c.String("dir"))
```

Comment:
I would move this call in the `else` block to avoid any possibility of having it yield an error when using `--all`

Reply:
Moved 🙂